### ISSUE TITLE
Major rewrite. Bootstrap, middleman, build process update

### ISSUE THREAD

**Issue by ShmoogleOsukami** [gridcoin-community/Gridcoin-Site#152]
# This issue documents the changes I'm attempting to make

I was going to do one big PR but I'm now going to split it up into lots of smaller PRs. Since that's the whole point of github and source control xD

## First major change
The first major change is updating Middleman and the associated build instruction that accompany it.

Middleman will now run on the latest version of 4.3.3, Bundler is now running version 2.0.1, Other dependences have been updated as well.

Readme.MD has been updated to include the commands needed to get middleman up and running.
It now specify specific version so the instruction work far into the future. 
It assumes a clean Linux install, some problems may arise on a dirty install that might have multiple gem versions etc. I'll be testing this out later on.

When it comes to making the PR I'll be providing instructions on how to deploy to the webserver as it can be a bit messy when not working from a blank slate.

## Second major change

The second major change is updating bootstrap to version 4.3.1 and removing Now-UI-Kit, easy tables, tether and other under used addons.

The reason I opted to do this is because bootstrap alone is enough to maintain the sites current look and even enhance it in places. This lets us offload a number of css files and other file loads, Most browsers only download 2-5 files at any one time when fetching css, js etc. So the less we reference the better.

## Third major change

The Third major change is generally going over all the HTML, updating css and class attribute to use bootstrap and fixing any oddities and problems, I also plan to update the whitelist to display compatible hardware and OS etc, much like the wiki whitelist. There will also be update to the guide and wording in the guides. So there will be a lot to review once this is done.

## (rough)Change log:

* Update middleman and build process.
* Added middleman-livereload to config.rb so it actually functions.
* Updated Bootstrap to version 4.3.1
* Updated Jquery to 3.3.1
* Bootstrap, Jquery and popper now use CDN sources.

* Removed tables.css
* Removed easyTables.css
* Removed Now-UI-Kit
* Removed Tether.JS
* Added Open Iconic Lib

* Rewrote CSS and HTML class attributes to better utilize Bootstrap
* Greatly improved responsiveness for mobile platforms
* Improved the styling in various places
* Updated Sync Wallet & Acquire GRC Guide
* Partialized many parts of the guides as information was shared in a lot of places.

**Comment by barton2526** [comment]
I guess the easiest way is to just give it a whirl and see what blows up

**Comment by barton2526** [comment]
Things that stand out from changelogs:

- Deprecated .text-hide—you'll see a warning during compilation—as it's a dated and undocumented feature. (4.1)
- Inline SVGs now have the same vertical alignment as images (4.1.2)
- Fixed: Silenced bad selectors in our JS plugins for the href HTML attribute to avoid JavaScript errors. Please try to use valid selectors or the data-target HTML attribute/target option where available. (4.3)
- Deprecated: img-retina, invisible, float, and size mixins are now deprecated and will be removed in v5. (4.3)

**Comment by ShmoogleOsukami** [comment]
Issue is not really relevant anymore as I've broken this down into easier to manage things.
